paimon-format/src/main/java/org/apache/paimon/format/avro/AvroFileFormat.java

```diff
@@ -62,11 +62,14 @@ public class AvroFileFormat extends FileFormat {
     public static final ConfigOption<Map<String, String>> AVRO_ROW_NAME_MAPPING =
             ConfigOptions.key("row-name-mapping").mapType().defaultValue(new HashMap<>());
 
-    private final FormatContext context;
+    private final Options options;
+    private final int zstdLevel;
 
     public AvroFileFormat(FormatContext context) {
         super(IDENTIFIER);
-        this.context = context;
+
+        this.options = getIdentifierPrefixOptions(context.options(), false);
```

Comment:
Can you also modify `AVRO_OUTPUT_CODEC` and `AVRO_ROW_NAME_MAPPING` to add prefix?